study.md

```diff
@@ -54,5 +64,5 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+streams process items one at a time, similar to enumerable, functions don't operate on them as a whole but affect each object.
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.